dbfit-java/core/src/main/java/dbfit/api/PreparedDbCommand.java

```diff
@@ -0,0 +1,18 @@
+package dbfit.api;
+
+import java.sql.SQLException;
+
+/**
+ * Represents a prepared database statement with ability to bind input parameters and
+ * to register and access output ones including return values.
+ */
+public interface PreparedDbCommand extends DbCommand, DbQuery {
```

Comment:
By including `Prepared` in the name do we tie ourselves to the JDBC `PreparedStatement` or reveal clues about implementation details? 

Would something like `ParameterisedDbCommand` be an alternative?